routes.go

```diff
@@ -2,15 +2,33 @@ package main
 
 import (
 	"budget-helper/database"
+	"budget-helper/users"
 	"fmt"
 	"log"
 	"net/http"
 )
 
-func home(db *database.Database, w http.ResponseWriter, r *http.Request) {
+func handleHome(db *database.Database, w http.ResponseWriter, r *http.Request) {
 	_, err := fmt.Fprint(w, "Welcome to Budget Helper!\n")
 	if err != nil {
-		log.Panic(err)
+		log.Fatalf("ERROR: %v\n", err)
+	}
+}
+
+func handleUsers(db *database.Database, w http.ResponseWriter, r *http.Request) {
+	_, err := fmt.Fprint(w, "Fetching users...\n")
+	if err != nil {
+		log.Fatalf("ERROR: %v\n", err)
+	}
+
+	repo := users.NewUserRepo(db)
```

Comment:
This will do for now, but newing up and entire repo every request is extremely expensive.

Reply:
Should all repos be initialized in the same place somewhere?

Reply:
Yes, we'll refactor to the structure you can find in my repo:

```
type Server struct {
    SomeRepo *SomeRepo
    Logger *log.Logger
}
```

Reply:
Next PR that is.